content/tutorials/get-started/get-started-using-php.md

```diff
@@ -0,0 +1,288 @@
+                (
+                    [Account] => rDTRjR6sWrRmGe18KMVwBuL212gLpArVLy
+                    [Balance] => 10000000000
+                    [Flags] => 0
+                    [LedgerEntryType] => AccountRoot
+                    [OwnerCount] => 0
+                    [PreviousTxnID] => AE18C0B30DE740490E66E92D9F45162C8860A6D9FCF279CF7A51FAFE05F573FB
+                    [PreviousTxnLgrSeq] => 42719256
+                    [Sequence] => 42719256
+                    [index] => 4DD9F7FAE7365B7A917932D6453DBA9B223AA4FF7193691EF6E5EE230519F4CA
+                )
+
+            [account_flags] => Array
+                (
+                    [defaultRipple] => 
+                    [depositAuth] => 
+                    [disableMasterKey] => 
+                    [disallowIncomingCheck] => 
+                    [disallowIncomingNFTokenOffer] => 
+                    [disallowIncomingPayChan] => 
+                    [disallowIncomingTrustline] => 
+                    [disallowIncomingXRP] => 
+                    [globalFreeze] => 
+                    [noFreeze] => 
+                    [passwordSpent] => 
+                    [requireAuthorization] => 
+                    [requireDestinationTag] => 
+                )
+
+            [ledger_hash] => 57F0CB8311CDEAE9AC60854CC482990CE971D1FD36DB1CC215B7A6634E27E739
+            [ledger_index] => 42719256
+            [status] => success
+            [validated] => 1
+        )
+
+    [warnings:protected] => 
+    [status:protected] => success
+    [type:protected] => response
+)
+```
+
+#### Interpreting the response
+
+The response fields contained in `AccountInfoResponse` that you want to inspect in most cases are:
+
+* `['account_data']['Sequence']` — This is the sequence number of the next valid transaction for the account. You need to specify the sequence number when you prepare transactions.
+
+* `['account_data']['Balance']` — This is the account's balance of XRP, in drops. You can use this to confirm that you have enough XRP to send (if you're making a payment) and to meet the [current transaction cost](transaction-cost.html#current-transaction-cost) for a given transaction.
+
+* `['validated']` — Indicates whether the returned data is from a [validated ledger](open-closed-validated-ledgers.html). When inspecting transactions, it's important to confirm that [the results are final](finality-of-results.html) before further processing the transaction. If `validated` is `true` then you know for sure the results won't change. For more information about best practices for transaction processing, see [Reliable Transaction Submission](reliable-transaction-submission.html).
+
+For a detailed description of every response field, see [account_info](account_info.html#response-format).
+
+
+## Keep on building
+
+Now that you know how to use `xrpl4j` to connect to the XRP Ledger, get an account, and look up information about it, you can also use `xrpl4j` to:
```

Comment:
Couple mentions of `xrpl4j` here that should probably be changed to be `XRPL_PHP`.

```suggestion
Now that you know how to use `XRPL_PHP` to connect to the XRP Ledger, get an account, and look up information about it, you can also use `XRPL_PHP` to:
```